src/app/routes/learnerRoutes.js

```diff
@@ -47,6 +47,29 @@ module.exports = function (app) {
     })
   )
 
+  app.delete('/learner/user/v1/delete/:userId',
+  // proxyUtils.verifyToken(),
+   isAPIWhitelisted.isAllowed(),
+  proxy(envHelper.learner_Service_Local_BaseUrl, {
+    proxyReqOptDecorator: proxyUtils.decorateRequestHeaders(envHelper.learner_Service_Local_BaseUrl),
+    proxyReqPathResolver: (req) => {
+      return require('url').parse(envHelper.LEARNER_URL + req.originalUrl.replace('/learner/', '')).path
+    },
+    userResDecorator: (proxyRes, proxyResData, req, res) => {
+      logger.info({ msg: '/learner/user/v1/delete called upstream url /user/v1/delete' });
+      try {
+        const data = JSON.parse(proxyResData.toString('utf8'));
+        if (req.method === 'DELETE' && proxyRes.statusCode === 404 && (typeof data.message === 'string' && data.message.toLowerCase() === 'API not found with these values'.toLowerCase())) res.redirect('/')
```

Comment:
Only check for statusCode here for failures..
Are we handling only 403?

Remove other check req.method, data.message etc.